modin/config/envvars.py

```diff
@@ -446,6 +446,13 @@ def get(cls):
         return min_partition_size
 
 
+class TestReadFromSqlServer(EnvironmentVariable, type=bool):
```

Comment:
Perhaps we need a test-specific configuration. Can be done in a different PR.  cc @YarShev